### PR TITLE
Append mash screen hits to assembly fasta

### LINKED ISSUE #22: Add filterscreen and appendscreen local modules to workflow
### Description of feature

- [x] Install append_screen_hits modules using `nf-core modules create ...`
- [x] Add local module to `workflows/phageannotator.nf`
- [x] Use `MASH_SCREEN.out.screen`, `params.mash_screen_reference_fasta`, and `ch_input.fasta` as input to APPEND_SCREEN_HITS
- [x] Update `nextflow.config` to include local module parameters (mash_screen_min_score) 
- [x] Update `nextflow_schema.json` to include local params using `nf-core schema build`
- [x] Update `conf/modules.config` to include local extra arguments
- [x] Test local modules using `nextflow run . -profile test,singularity --outdir results`
- [x] Add nf-test for local module
- [x] Update nf-tests for workflow
- [x] Make sure nf-tests all pass
- [x] Lint additions using `nf-core lint`
- [x] Submit a PR mash modules

Image

### PR DESCRIPTION
**Pull request by CarsonJM**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/phageannotator/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/phageannotator _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by CarsonJM** [comment]
@nf-core-bot fix-linting

File: workflows/tests/phageannotator.nf.test.snap [hunk 1]
```diff
@@ -1,22 +1,22 @@
 {
```
Comment (sateeshperi) [inline workflows/tests/phageannotator.nf.test.snap:1]: where is the `phageannotator.nf.test` file?

Comment (CarsonJM) [reply workflows/tests/phageannotator.nf.test.snap:1]: Forgot to update it initially. Updated phageannotator.nf.test and snap file now included

File: modules/local/append_screen_hits/main.nf [hunk 2]
```diff
@@ -0,0 +1,54 @@
+process APPEND_SCREEN_HITS {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "conda-forge::biopython=1.78 conda-forge::pandas=1.3.5"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-80c23cbcd32e2891421c54d1899665046feb07ef:77a31e289d22068839533bf21f8c4248ad274b60-0' :
+        'quay.io/biocontainers/mulled-v2-80c23cbcd32e2891421c54d1899665046feb07ef:77a31e289d22068839533bf21f8c4248ad274b60-0' }"
```
Comment (sateeshperi) [inline modules/local/append_screen_hits/main.nf]:
```suggestion
        'biocontainers/mulled-v2-80c23cbcd32e2891421c54d1899665046feb07ef:77a31e289d22068839533bf21f8c4248ad274b60-0' }"
```

the registries are provided separately in `nextflow.config` now. could you double check this?

Comment (CarsonJM) [reply modules/local/append_screen_hits/main.nf]: I just removed the prefix and the tests still passed!

**Review by sateeshperi — APPROVED** [review]
lgtm

**Pull request merged**